apps/web-client/src/lib/components/Tables/InventoryTables/BookPriceCell.svelte

```diff
@@ -9,12 +9,12 @@
 {#if price && warehouseDiscount}
 	<div class="flex flex-col items-start gap-0.5">
 		<span class="sr-only">Discounted price:</span>
-		<span>€{(price - warehouseDiscount / 100).toFixed(2)}</span>
+		<span>€{((price * (100 - warehouseDiscount)) / 100).toFixed(2)}</span>
 		<span class="sr-only">Original price:</span>
-		<span class="text-gray-400 line-through">(€{price})</span>
+		<span class="text-gray-400 line-through">(€{price.toFixed(2)})</span>
 		<span class="sr-only">Percentage discount:</span>
 		<span class="text-gray-400">-{warehouseDiscount}%</span>
 	</div>
 {:else}
-	{price ? `€${price}` : "N/A"}
+	{price ? `€${price.toFixed(2)}` : "N/A"}
```

Comment:
Can you refactor this so that:
- if there's only price, the price is shown in a span element with `data-property="full-price"`
- if the discount is also present, the same logic is used:
  - full price is a span with `data-property="full-price"`
  - discount is a span with `data-property="applied-discount"`
  - discounted price is a span with `data-property="discounted-price"`
  
  That way we could also update the price element in e2e api for a more fine grained control (and avoid the full string match)